### ISSUE TITLE
For Job Seekers: Add a legend element to the fieldset, or remove the fieldset element

### ISSUE THREAD

**Issue by webinista** [department-of-veterans-affairs/veterans-employment-center#314]
[URL of the issue](https://www.vets.gov/veterans-employment-center/for_job_seekers)

[URL of the report](http://vawsb.dequecloud.com/FireEyes/page/show/2140808)

HTML block begins with: 
`<fieldset class="borderless"> <div class="row"> <div class="small-12 columns">`

`fieldset` should be used to group related fields within a form, not an entire form. If it's just a case of a missing legend, add it. Otherwise remove `fieldset`.

**Comment by webinista** [comment]
Old enough that it's probably irrelevant.
